presto/tests/conftest.py

```diff
@@ -17,4 +18,9 @@ def dd_environment(instance):
 @pytest.fixture(scope='session', autouse=True)
 @pytest.mark.usefixtures('dd_environment')
 def instance():
-    return load_jmx_config()
+    inst = load_jmx_config()
+    # Add presto coordinator to the configuration
+    inst.get('instances').append(deepcopy(inst.get('instances')[0]))
```

Comment:
it looks like you are appending to inst['instances'] a copy of something that is already there 😕

Reply:
Yes, actually the environment started by `ddev env start` has 3 nodes: 1 coordinator and 2 workers. The configuration file needs to have 1 instance of worker and 1 instance of coordinator in order to have all the metrics.

However `load_jmx_config` returns a configuration with only 1 instance (worker). So this just adds another instance to the configuration, corresponding to the coordinator. These instances are the same except for the port.